sklearn/preprocessing/_data.py

```diff
@@ -1468,14 +1544,13 @@ def get_feature_names(self, input_features=None):
 
         Parameters
         ----------
-        input_features : list of string, length n_features, optional
+        input_features : list of str, length n_features, default=None
             String names for input features if available. By default,
             "x0", "x1", ... "xn_features" is used.
 
         Returns
         -------
-        output_feature_names : list of string, length n_output_features
-
+        output_feature_names : list of str, length n_output_features
```

Comment:
```suggestion
        output_feature_names : list of str of shape (n_output_features,)
```

Reply:
Thanks for the suggestions. I updated the PR!